api/observability/v1/output_types.go

```diff
@@ -305,7 +305,7 @@ type Cloudwatch struct {
 	// +nullable
 	Tuning *CloudwatchTuningSpec `json:"tuning,omitempty"`
 
-	// +kubebuilder:validation:Required
```

Comment:
Is required for our config.   Is only optional if they already have an env variable set (which they dont).

Reply:
The client has no notion of a Default?

Reply:
I guess maybe not based on their example? https://github.com/awslabs/aws-sdk-rust/blob/main/examples/examples/cloudwatchlogs/src/bin/list-log-groups.rs#L44